.github/workflows/php_tests.yml

```diff
@@ -0,0 +1,65 @@
+name: PHP Tests
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+  workflow_dispatch:
+
+jobs:
+  php_lint:
+    name: PHP Syntax Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout latest WCS
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Switch to PHP 7.4
+        run: |
+          sudo update-alternatives --set php /usr/bin/php7.4
+          sudo update-alternatives --set phar /usr/bin/phar7.4
+          sudo update-alternatives --set phar.phar /usr/bin/phar.phar7.4
+      - name: Lint
+        run: find . -type f -name "*.php" -exec php -l {} \;
+  php_build_and_test:
+    name: PHP Unit Test (WP, WC)
+    runs-on: ubuntu-latest
+    services:
+      mariadb:
+        image: mariadb:latest
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_USER: wp_test
+          MYSQL_PASSWORD: wp_test
+          MYSQL_DATABASE: wordpress_default
+          MYSQL_ROOT_PASSWORD: root
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+    strategy:
+      matrix:
+        wp-version: [5.4.4, 5.5.3, latest]
+        wc-version: [4.8.0, 4.9.2, 5.0.0]
+    steps:
+      - name: Checkout latest WCS
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Switch to PHP 7.4
+        run: |
+          sudo update-alternatives --set php /usr/bin/php7.4
+          sudo update-alternatives --set phar /usr/bin/phar7.4
+          sudo update-alternatives --set phar.phar /usr/bin/phar.phar7.4
+      - name: Install composer
+        run: |
+          php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+          php -r "if (hash_file('sha384', 'composer-setup.php') === '756890a4488ce9024fc62c56153228907f1545c228516cbf63f885e036d37e9a59d27d63f46af1d4d07ee0f76181c7d3') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
```

Comment:
Why include the hash check? When a new version of `composer-setup` is released, wouldn't this break?

Reply:
I copied the download script directly from https://getcomposer.org/download/. 

> When a new version of composer-setup is released, wouldn't this break?

This is a valid concern. Do we care about the file integrity/signing here? Or should we manually update it when it breaks?

Reply:
Maybe I should do this instread: https://getcomposer.org/doc/faqs/how-to-install-composer-programmatically.md

Reply:
> This is a valid concern. Do we care about the file integrity/signing here? Or should we manually update it when it breaks?

I wouldn't be interested about the new version. I think I would find cumbersome to update it at every version, while nothing can get merged until a PR is opened to fix the hash in the CI configuration

Reply:
Fixed. Now using a script to download latest composer and also check its signature.